client/state/reader/posts/normalization-rules.js

```diff
@@ -96,7 +97,7 @@ export function classifyPost( post ) {
 		displayType ^= DISPLAY_TYPES.FEATURED_VIDEO;
 	}
 
-	if ( post.content_images && filter( post.content_images, imageIsBigEnoughForGallery ).length >= GALLERY_MIN_IMAGES ) {
+	if ( post.content_images && imagesForGallery.length >= GALLERY_MIN_IMAGES ) {
```

Comment:
Good plan. I'll simplify now.

I wonder if we could also drop the logic for `LARGE_BANNER`, `LANDSCAPE_BANNER`, `PORTRAIT_BANNER` and `THUMBNAIL`? We don't appear to be using those anywhere. cc @blowery